sonar-db/src/main/resources/org/sonar/db/user/UserMapper.xml

```diff
@@ -98,6 +98,14 @@
     </where>
   </select>
 
+  <select id="countByEmail" parameterType="String" resultType="long">
+    SELECT count(u.id)
+    FROM users u
+    <where>
```

Comment:
Emails are not case-sensitive, so I suppose a hack to insert duplication is to type an email with different case. @eric-duquesnoy how can such constraint be addressed ?